CHANGELOG.md

```diff
@@ -627,7 +629,11 @@ authentication credentials in Kong.
   releases of Kong since every release of Kong is introducing breaking changes
   to the Admin API.
 
+<<<<<<< HEAD
 [Unreleased]: https://github.com/Kong/go-kong/compare/v0.33.0...Unreleased
+=======
+[v0.34.0]: https://github.com/Kong/go-kong/compare/v0.33.0...v0.34.0
+>>>>>>> e09693a (feat: add support to consumer_groups)
```

Comment:
Rebase went awry:
```suggestion
[v0.34.0]: https://github.com/Kong/go-kong/compare/v0.33.0...v0.34.0
```
The `...Unreleased` that's currently there isn't any more valid than `...v034.0`; we should use the latter since these are more placeholders to make the link work once the tag exists (since we often forget them at release time).

Reply:
Ops..fixed now.

> we should use the latter since these are more placeholders to make the link work once the tag exists

Noted!